website/docs/Queries.md

```diff
@@ -199,6 +203,90 @@ const { getByA11yValue } = render(<Component />);
 const element = getByA11yValue({ min: 40 });
 ```
 
+## TextMatch
+
+Some APIs accept an object containing options affecting precision of string matching as the last argument:
+
+```typescript
+type TextMatchOptions = {
+  exact?: boolean;
+  normalizer?: (textToNormalize: string) => string;
+  trim?: boolean;
+  collapseWhitespace?: boolean;
+};
+```
+
+`exact` option defaults to `true` but if you want to search for a text slice or make text matching case-insensitive you can override it. That being said we advise you to use regex in more complex scenarios.
+
+### Normalization
+
+Before running any matching logic against text, it is automatically normalized. By default, normalization consists of trimming whitespace from the start and end of text, and collapsing multiple adjacent whitespace characters into a single space.
+
+If you want to prevent that normalization, or provide alternative normalization (e.g. to remove Unicode control characters), you can provide a `normalizer` function in the options object. This function will be given a string and is expected to return a normalized version of that string.
+
+:::info
+Specifying a value for `normalizer` replaces the built-in normalization, but you can call `getDefaultNormalizer` to obtain a built-in normalizer, either to adjust that normalization or to call it from your own normalizer.
```

Comment:
you can't call `getDefaultNormalizer ` because it's not exported from the library entrypoint